libpldmresponder/bios_config.cpp

```diff
@@ -447,9 +447,11 @@ void BIOSConfig::updateBaseBIOSTableProperty()
                                           dbusProperties, "Set");
         std::variant<BaseBIOSTable> value = baseBIOSTableMaps;
 #ifdef OEM_IBM
+        std::string dirPath = "/var/lib/pldm/bootSide";
+        const fs::path bootSideDirPath = dirPath;
```

Comment:
we can use a single statment to do the fs path initialisation, first declaring string and then converting to fs:path object is redundant